mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafText.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LeafText.swift
+//  
+//
+//  Created by Cris Messias on 06/10/23.
+//
+
+import SwiftUI
+
+//MARK: - TitleModifier
+
+public struct TitleModifier: ViewModifier {
```

Comment:
I think it does not need to be public. This way we ensure this will always be used via `View` extension instead of calling `modifier`.

Reply:
I tried to use it without the public keyword but the modifiers are inaccessible in other views inside the project. Do you have any idea how to solve this?  By the way, if I take self from the extension Xcode gives me errors.

//MARK: - TitleModifier
/public extension Text {
    func title() -> some View {
        @Environment(\.leafTheme)  var theme
        return self
            .font(theme.font.titleLarge)
            .bold()
            .lineSpacing(theme.size.lineSpacingTitle)
            .foregroundStyle(theme.color.content.primary)
    }
}


//MARK: - SubtitleModifier
public extension Text {
    func subtitle() -> some View {
        @Environment(\.leafTheme)  var theme
        return self
            .font(theme.font.subtitle)
            .bold()
            .foregroundStyle(theme.color.content.secondary)
    }
}

//MARK: - BodyModifier
public extension Text {
    func body() -> some View {
        @Environment(\.leafTheme)  var theme
        return self
            .font(theme.font.body)
            .foregroundStyle(theme.color.content.secondary)
    }
}

//MARK: - LabelModifier
public extension Text {
    func label() -> some View {
        @Environment(\.leafTheme)  var theme
        return self
            .font(theme.font.label)
            .bold()
            .foregroundStyle(theme.color.content.tertiary)
    }
}

Reply:
Ok, maybe it needs to be public. Keep it this way! However, that is always a good practice: make public only what really needs to be public.

Reply:
Okay! I finished!